### DISCUSSION TITLE
Release 2.11.1 missing

### DISCUSSION THREAD

**Discussion by roperscrossroads** [nextcloud/helm#183] [Q&A] [answered]
Hi,

Release 2.11.1 was removed from the releases page. Is there a reason why it was deleted? I am using Jenkins X 3 and my pipelines fail now because it cannot access the file.

Thanks!

Adam Roper

**Comment by tvories** [comment, marked as answer]
That release had some issues and was removed.  Were you able to fix your pipelines?

**Reply by roperscrossroads** [reply]
> That release had some issues and was removed. Were you able to fix your pipelines?

@tvories 

I found the tgz file in the "tags" section and set up my own helm chart repo to get around it.

I would suggest that you not delete releases - maybe just edit the release info instead if there is a problem. With Jenkins X, we could not release any other software until the system had access to nextcloud-2.11.1.tgz. I'm glad it was still in the tags section!

Thanks!
